common/src/main/scala/com/gu/zuora/model/response/Responses.scala

```diff
@@ -34,12 +34,21 @@ object ZuoraErrorResponse {
 case class ZuoraErrorResponse(success: Boolean, errors: List[ZuoraError])
     extends Throwable(errors.asJson.spaces2) with ZuoraResponse {
 
+  override def toString: String = this.errors.toString()
+  def toRetryNone: RetryNone = new RetryNone(message = this.asJson.noSpaces, cause = this)
+  def toRetryUnlimited: RetryUnlimited = new RetryUnlimited(this.asJson.noSpaces, cause = this)
+
+  // Based on https://knowledgecenter.zuora.com/DC_Developers/G_SOAP_API/L_Error_Handling/Errors#ErrorCode_Object
   def asRetryException: RetryException = errors match {
-    case List(ZuoraError("UNKNOWN_ERROR", _)) => new RetryUnlimited(this.asJson.noSpaces, cause = this)
-    case _ => new RetryNone(message = this.asJson.noSpaces, cause = this)
+    case List(ZuoraError("API_DISABLED", _)) => toRetryUnlimited
```

Comment:
Hopefully this never happens, but if it does we'll now have 24 hours to fix it before we lose the contributors!